ledger/src/blockstore.rs

```diff
@@ -7445,7 +7445,7 @@ pub mod tests {
     #[test]
     fn test_insert_multiple_is_last() {
         solana_logger::setup();
-        let (shreds, _) = make_slot_entries(0, 0, 19, /*merkle_variant:*/ true);
+        let (shreds, _) = make_slot_entries(0, 0, 18, /*merkle_variant:*/ true);
```

Comment:
I'm guessing you're reducing number of entries to keep number of shreds returned static?
If I have that right, why are we now getting more shreds?

Reply:
i think now that we add the retransmit signature, there's less space for entries, so it requires more shreds for the same amount of entries.